src/invidious/channels/about.cr

```diff
@@ -29,29 +29,14 @@ struct AboutRelatedChannel
 end
 
 def get_about_info(ucid, locale)
-  result = YT_POOL.client &.get("/channel/#{ucid}/about?gl=US&hl=en")
-  if result.status_code != 200
-    result = YT_POOL.client &.get("/user/#{ucid}/about?gl=US&hl=en")
+  begin
+    initdata = request_youtube_api_browse(browse_id: ucid, params: "")
```

Comment:
`params` should be equal to `EgVhYm91dA==` (protobuf object to get the "about" page), no?

Reply:
Great idea, this reduced the received data from 1660KB to 98KB